plugins/inputs/nsq_consumer/README.md

```diff
@@ -0,0 +1,26 @@
+# NSQ Consumer Input Plugin
+
+The [NSQ](http://nsq.io/) consumer plugin polls a specified NSQD
+topic and adds messages to InfluxDB. The plugin assumes messages follow the
```

Comment:
it doesn't assume this, it uses the format specified by `data_format`, which can be any of the supported input data formats.